### PR TITLE
pulp_pull: make pulls insecure

### PR DESCRIPTION
**Pull request by vrutkovs**

### CODE REVIEW

File: atomic_reactor/plugins/post_pulp_pull.py [hunk 1]
```diff
@@ -59,7 +59,7 @@ def run(self):
 
         while True:
             # Pull the image from Crane
-            name = self.tasker.pull_image(pullspec)
+            name = self.tasker.pull_image(pullspec, insecure=True)
```
Comment (lcarva) [inline atomic_reactor/plugins/post_pulp_pull.py]: Would be nice to have this configurable.

Comment (vrutkovs) [inline atomic_reactor/plugins/post_pulp_pull.py]: Agree, a config option is a good idea. Which default value should we stick to - I guess `insecure=False`?

Comment (lcarva) [inline atomic_reactor/plugins/post_pulp_pull.py]: Yes, I think that would be in line with how the other plugins behave. There's probably also an existing way to determine this value from osbs-client for when configuring this plugin.

**Comment by vrutkovs** [comment]
Test case added

**Pull request merged**